code/modules/mob/living/carbon/human/species/abductor.dm

```diff
@@ -24,4 +24,7 @@
 	H.gender = NEUTER
 	if(H.mind)
 		H.mind.abductor = new /datum/abductor
-	return ..()
\ No newline at end of file
+	H.remove_language("Tradeband")
```

Comment:
Because there's realllly no other way.

It's either copy-pasting these same three things into the abductors game mode for after the abductor gets made...orrr this. There's only two procs for adding/removing languages, and that's it.

It's not like body accessories where you can just null it out and be done with it.